pom.xml

```diff
@@ -206,7 +206,7 @@
                 <plugin>
                     <groupId>org.openapitools</groupId>
                     <artifactId>openapi-generator-maven-plugin</artifactId>
-                    <version>4.1.2</version>
+                    <version>4.1.3</version>
```

Comment:
I think we might be able to remove the `jackson-databind` `dependencyManagement` section. I'll investigate.